storybook/src/types.ts

```diff
@@ -31,6 +41,14 @@ export type SymfonyOptions = {
      * Additional paths to watch during compilation.
      */
     additionalWatchPaths?: string[];
+
+    /**
+     * When Storybook and Symfony are not on the same host (e.g. Symfony is Dockerized), mounted paths may differ.
+     * This option allows to map paths from Symfony host to Storybook host.
+     */
+    templatePathAliases?: {
+        [p: string]: string;
+    };
```

Comment:
As my Symfony app is dockerized, my host path differs from mounted path

* On machine: `/path/to/project/front`
* On Docker images: `/application/front`

This allows me to have this

```js
templatePathAliases: {
    [path.resolve(__dirname, '..')]: '/application/front',
}
```

Reply:
Note: changed `templatePathAliases` to `projectPathAliases`

Reply:
Not required anymore